WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorView.swift

```diff
@@ -151,9 +151,7 @@ struct ProductSelectorView: View {
         if let variationListViewModel = viewModel.getVariationsViewModel(for: rowViewModel.productOrVariationID) {
             HStack {
                 ProductRow(multipleSelectionsEnabled: configuration.multipleSelectionsEnabled,
-                           viewModel: rowViewModel) {
-                    viewModel.toggleSelectionForVariations(of: rowViewModel.productOrVariationID)
-                }
+                           viewModel: rowViewModel)
```

Comment:
I missed this change in the last review 😅 I think for coupons we want to be able to select all variations for a product (to include or exclude them from a coupon) - so this callback should not be removed.

We should add another config for the `ProductSelectorViewModel` to indicate whether this is the behavior or not with something like `enableTogglingSelectionForAllVariations`.

Reply:
I see what you mean, somehow I missed this behaviour too. I made the changes here: https://github.com/woocommerce/woocommerce-ios/pull/8935/commits/bc6e1c49004c03774fe985b277d5ef3c7c3ecfa9 , leaving the toggle for all variations to true by default and disallowing it in the ProductSelector.